src/commands/trialApp/removeTrialApp.ts

```diff
@@ -3,26 +3,34 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { MessageItem } from 'vscode';
+import { commands, MessageItem } from 'vscode';
 import { IActionContext } from 'vscode-azureextensionui';
 import { TrialAppContext } from '../../constants';
+import { ITrialAppContext } from '../../explorer/trialApp/ITrialAppContext';
 import { TrialAppTreeItem } from '../../explorer/trialApp/TrialAppTreeItem';
 import { ext } from '../../extensionVariables';
 import { localize } from '../../localize';
 import { addTrialAppTelemetry } from './addTrialAppTelemetry';
 
 export async function removeTrialApp(context: IActionContext, node?: TrialAppTreeItem): Promise<void> {
     if (!node) {
-        node = await ext.tree.showTreeItemPicker<TrialAppTreeItem>(TrialAppTreeItem.contextValue, context);
+        node = ext.azureAccountTreeItem.trialAppNode;
     }
 
-    const message: string = localize('removeTrialApp', 'Are you sure you want to remove trial app "{0}"?', node.client.fullName);
+    const trialAppContext: ITrialAppContext | undefined = ext.context.globalState.get(TrialAppContext);
+    if (!trialAppContext) {
+        throw new Error(localize('noTrialContext', 'Cannot get trial app context.'));
+    }
+
+    const message: string = localize('removeTrialApp', 'Are you sure you want to remove trial app "{0}"?', trialAppContext.name);
```

Comment:
Could you go back and make those changes we discussed in the other PR?

Reply:
Since we are now able to call removeTrialApp from the command palette, we no longer have access to the trial app node. So I think the best way is to grab the name from here instead of using a tree item picker (since there is always only 1 trial app)